### PR TITLE
Instructions for Overleaf

### PR DESCRIPTION
**Pull request by johdvos**
This PR contains instructions for the use of the template in Overleaf.

### CODE REVIEW

**Comment by johdvos** [comment]
@guacke if you agree with these instructions. I will notify the people responsible of distributing this template.

**Review by guacke — APPROVED** [review]
👍

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 <img src="media/writing.png" width="400">
 </p>
 
+This article template can be used in Overleaf using the link below. We have also added [instructions](instructions.md) on how to use this template in overleaf.
```
Comment (johdvos) [inline README.md:5]:
```suggestion
This article template can be used in Overleaf using the link below. We have also added [instructions](instructions.md) on how to use this template in Overleaf.
```

File: instructions.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+- Click on the Overleaf link on the [home GitHub page](README.md). An Overleaf project will open with the article template as shown below. Note that this is a read-only project and you cannot yet make changes.
```
Comment (johdvos) [inline instructions.md:1]:
```suggestion
- Click on the Overleaf link on the [home GitHub page](README.md). An Overleaf project will open with the article template, as shown below. Note that this is a read-only project and you cannot yet make changes.
```

File: instructions.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+<img src="media/overleaf_project_RO.png" width="400">
+</p>
+
+- Click on the *Overleaf menu* button in the top left corner (indicated by the red arrow).
+
+<p align="center">
+<img src="media/overleaf_project_RO_annotated.png" width="400">
+</p>
+
+- The overleaf menu opens at the left of the screen. Next, click on *copy project* under the tab *Actions* to copy the template to an editable project.
```
Comment (johdvos) [inline instructions.md:13]:
```suggestion
- The Overleaf menu opens at the left of the screen. Next, click on *copy project* under the tab *Actions* to copy the template to an editable project.
```

**Pull request merged**